### ISSUE TITLE
Prefilter pVACseq and pVACsplice on a user-defined list of biotypes

### ISSUE THREAD

**Issue by susannasiebert** [griffithlab/pVACtools#1090]
Currently, pVACseq does not prefilter transcripts on their biotype. We prioritize protein_coding transcripts when picking the best peptide and transcript in the aggregate report creation.

In pVACsplice we hard-pre-filter on protein_coding transcripts.

Instead we should define a new parameter `--biotypes` with a default of `['protein_coding']` that applies a prefilter on transcript biotypes both in pVACseq and pVACsplice. The aggregate report will continue to prioritize protein_coding transcripts when selecting the best peptide/transcripts. The docs should explain that "a few other, more speculative types (e.g. `non_stop_decay` and `nonsense_mediated_decay`) can give rise to neoantigens if users want to include them, they can specify their own list of biotypes".

**Comment by sufyazi** [comment]
hi I'm wondering when pVACsplice will be ready for use? I read your publication regarding pVACsplice but now I'm confused when I see that there is no pVACsplice in the pVACtools suite.

**Comment by susannasiebert** [comment]
Hi @sufyazi, we are currently doing extensive testing on this tool on various patient data and are hoping to release it before the end of the year. May I ask which publication you are referencing? While we've presented this tool at various conferences, we haven't published a full paper on it yet.

**Comment by sufyazi** [comment]
Hi Susanna, sorry if I was implying the wrong thing. I saw an abstract of pVacSplice here 10.1016/j.cancergen.2022.10.114 and wanted to find out more about it, only to realize it is not yet there in pVacTools. I just started using pVacTools at work for a new project we are embarking, and pVacSplice sounds perfect to incorporate into our pipeline, so I was just wondering.

Thank you for the update! I will look forward to it!
